docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -139,7 +139,8 @@ If your machine does not support modulefiles, you can instead run the provided b
 
 .. code-block:: console
 
-   chmod +x ${INSTALL_PREFIX}/bin/setenv_nceplibs.sh ${INSTALL_PREFIX}/bin/setenv_nceplibs.sh
+   chmod +x ${INSTALL_PREFIX}/bin/setenv_nceplibs.sh 
+   ${INSTALL_PREFIX}/bin/setenv_nceplibs.sh
```

Comment:
This script must be sourced, not executed, therefore it doesn't have the executable flags. Shouldn't the command be simply
```
source ${INSTALL_PREFIX}/bin/setenv_nceplibs.sh
```
?

Reply:
If you add the executable flags and simply run it, the settings are gone once the script finishes.

Reply:
@climbfuji This could explain the difference in behavior I've been seeing requiring manual setting of some variables. Let me try that out.

Reply:
Yes, it definitely needs to be sourced, not executed.

Reply:
Okay, @climbfuji the instructions have been updated. Can you remind me which extra python packages were needed for running the graphics script? I'll add those to this PR as well.

Reply:
These are all that I had to install:
```
pip3 install f90nml
pip3 install pyyaml
pip3 install jinja2
pip3 install pygrib
pip3 install cartopy
pip3 install matplotlib
pip3 install scipy
```
On Redhat 8, I got an error (see slack), for which I had to do:
```
# Fix incompatible shapely version
pip3 uninstall shapely
pip3 install shapely --no-binary shapely
```
but this goes into the issues/bugs document.

Reply:
Okay, I confirmed the same packages worked when installed through conda as well (I didn't need to manually install scipy since it was a prerequisite for cartopy, but it does't hurt to leave it in. I also needed to install the "pillow" package on my Mac (which was quite confusing, since it is not referred to as "pillow" when imported).

I think this PR is ready for merge, @JulieSchramm can you take a look to make sure my syntax changes won't break the readthedocs build?

Reply:
> Okay, I confirmed the same packages worked when installed through conda as well (I didn't need to manually install scipy since it was a prerequisite for cartopy, but it does't hurt to leave it in. I also needed to install the "pillow" package on my Mac (which was quite confusing, since it is not referred to as "pillow" when imported).
> 
> I think this PR is ready for merge, @JulieSchramm can you take a look to make sure my syntax changes won't break the readthedocs build?

Quite messy these different Python versions and ways to install. I had to install scipy explicitly on my mac with conda, but not pillow ...